src/main/webapp/data/userMapData.json

```diff
@@ -320,6 +329,7 @@
       "Danish Hydraulic Institute",
       "Erhvervsakademiet Lillebælt",
       "Hans Christian Ørsted Gymnasium",
+      "IBC",
```

Comment:
Is there a full name for this? We'd better avoid confusion :)

Reply:
Hi @dalessr , International Business Centre is the full name. I'll change it